### ISSUE TITLE
When updating NVDA, locale files are not removed

### ISSUE THREAD

**Issue by CyrilleB79** [nvaccess/nvda#20247]
Discovered in [this thread](https://groups.google.com/a/nvaccess.org/g/nvda-users/c/gfMSPCk7WVI) of the users mailing list.

Users may not have the same symbol file depending on if they have updated or cleanly installed NVDA. In the linked thread:
* Cyrille BOUGOT has updated accross beta versions
* FARHAN ISHRAK Fahim has probably cleanly installed NVDA 2026.1, or what is equivalent in this issue, updated from 2025.3.3 directly to 2026.1 (without installing beta versions) since 2026.1 series is the first NVDA to install in 64-bit programs folder.

### Steps to reproduce
* Download NVDA [2026.1beta1](https://download.nvaccess.org/releases/2026.1beta1/nvda_2026.1beta1.exe) and [2026.1.1rc1](https://download.nvaccess.org/releases/2026.1.1rc1/nvda_2026.1.1rc1.exe).
* In a folder named "CleanNVDA", install NVDA 2026.1.1rc1
* In another folder named "UpgradedNVDA", install NVDA 2026.1beta1
* Launch it and upgrade to last proposed version, i.e. 2026.1.1rc1
* In both NVDA install folders, look at `locale\bn\` subfolder
### Actual behavior
* In "UpdatedNVDA" folder, we have `locale\bn\symbols.dic`.
* In "CleanNVDA" folder, we have no `locale\bn\symbols.dic`.

### Expected behavior
The file structure should be the same, no matter if NVDA was upgraded or cleanly installed.

`locale\bn\symbols.dic` should have been removed in #19766, which was merged during beta phase.

### NVDA logs, crash dumps and other attachments
N/A
#### NVDA installed/portable/running from source
Tested with portable.

But also affects installed version as evidenced in [this thread](N/A) of NVDA mailing list.

#### NVDA version
2026.1.1rc1
#### Windows version
Windows 11 23H2 (AMD64) build 22631.6783
#### Name and version of other software in use when reproducing the issue
N/A
#### Other information about your system
N/A
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes (tested without add-on)
#### Does the issue still occur after you run the System Accessibility Repair Tool in NVDA's tools menu?
Not tested
